catalog-info.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elasticsearch-k8s-metrics-adapter
+  description: Buildkite Pipeline for elasticsearch-k8s-metrics-adapter
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elasticsearch-k8s-metrics-adapter
+
+spec:
+  type: buildkite-pipeline
+  owner: group:cloud-k8s-operator
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elasticsearch-k8s-metrics-adapter
+    spec:
+      repository: elastic/elasticsearch-k8s-metrics-adapter
+      pipeline_file: ".buildkite/pipeline.yml"
```

Comment:
You could add a simple `.buildkite/pipeline.yml` if you want to see the first builds green.

```sh
> echo 'steps:
  - command: echo "🐙"' > .buildkite/pipeline.yml
```